tests/test_conferences.py

```diff
@@ -368,6 +368,27 @@ def test_route_valid(self):
             assert_equal(msg.conference_name, 'conf')
             assert_equal(msg.conference_category, 'talk')
 
+    def test_alternate_route_valid(self):
+        conf = ConferenceFactory(endpoint='chocolate', active=True)
```

Comment:
Since you are not saving yet, you can use `ConferenceFactory.build()` here to avoid redundant database hits.
https://factoryboy.readthedocs.org/en/latest/#using-factories